chalicelib_fourfront/checks/helpers/wfr_utils.py

```diff
@@ -178,6 +178,8 @@
     'TrAC-loop':     ["HiC_Pipeline_0.2.6", "HiC_Pipeline_0.2.7"],
     # Preliminary - Released to network  # NO-NORM
     'PLAC-seq':      ["HiC_Pipeline_0.2.6", "HiC_Pipeline_0.2.7"],
+    # Preliminary - Released to network  # NO-NORM
```

Comment:
I do not understand why this diff is showing. This file is the same as what's in the master branch.